api/app/src/routes/medical/organization.ts

```diff
@@ -49,12 +58,19 @@ router.put(
   "/:id",
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
-    const orgId = getFromParamsOrFail("id", req);
-    const data = organizationSchema.parse(req.body);
+    const id = getFromParamsOrFail("id", req);
+    const payload = organizationUpdateSchema.parse(req.body);
 
-    const org = await updateOrganization({ id: orgId, cxId, data });
+    const updateCmd: OrganizationUpdateCmd = {
+      ...baseUpdateSchemaToCmd(payload),
```

Comment:
Convert from `eTag` to `version` - this is outstanding to try to remove by having the property on the model called `eTag`, optional, and string.